source/data-api.txt

```diff
@@ -52,298 +73,72 @@ The Data API supports two types of endpoints:
+- You can't or don't want to manage a MongoDB driver in your server-side
+  environment. For example, some edge compute environments don't support
+  database drivers or connection pooling.
 
-          { <content> }
+- You want to integrate Atlas data access into a federated API gateway
 
-   * - ``Double``
-     
-     - .. code-block:: json
-          :copyable: false
+- You want to connect to App Services from an environment not currently
+  supported by a Realm SDK and don't want to use a driver to connect
+  over the :ref:`wire protocol <wire-protocol>`.
 
-          {
-             "$numberDouble": "10.5"
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          10.5
-   
-   * - ``Int32``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {"$numberInt":"10"}
-     
-     - .. code-block:: json
-          :copyable: false
-
-          10
-   
-   * - ``Int64``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {"$numberLong":"50"}
-     
-     - .. code-block:: json
-          :copyable: false
-
-          50
-   
-   * - ``MaxKey``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          { "$maxKey": 1 }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {}
-       
-       *No JSON equivalent*
-
-   * - ``MinKey``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          { "$minKey": 1 }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {}
-       
-       *No JSON equivalent*
-
-   * - ``ObjectId``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "$oid":"5d505646cf6d4fe581014ab2"
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          "5d505646cf6d4fe581014ab2"
-
-   * - ``Regular Expression``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "$regularExpression": {
-                "pattern":"^H",
-                "options":"i"
-             }
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "Pattern": "^H",
-             "Options": "i"
-          }
-
-   * - ``Timestamp``
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "$timestamp": {
-                "t":1565545664,
-                "i":1
-              }
-          }
-     
-     - .. code-block:: json
-          :copyable: false
-
-          {
-             "T": 1565545664,
-             "I": 1
-          }
+An operation called through a Data API endpoint will likely take longer
+than the corresponding MongoDB operation called through a connected
+MongoDB Driver. For high-load use-cases and latency sensitive
+applications, we recommend connecting directly to your database with a
+MongoDB driver. To learn more, visit the :driver:`MongoDB Drivers
+</data-api-data-formats>` documentation.
```

Comment:
Another bullet that we might consider adding to this list – 

- "You are looking to create a new feature and would prefer starting your development on the client side and then moving to create / refine the API layer when your requirements are refined."